media-gfx/yafaray/yafaray-3.3.0-r2.ebuild

```diff
@@ -41,6 +40,7 @@ pkg_setup() {
 }
 
 src_prepare() {
+	append-flags "-pthread"
```

Comment:
That's flag-o-matic.

Reply:
ups, you're right... thought I saw this in toolchain-funcs sometime ago. Will fix it.